### PR TITLE
Introduce PATCH request into webservice API

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Documentation related to https://github.com/PrestaShop/PrestaShop/pull/27952

### CODE REVIEW

File: webservice/getting-started.md [hunk 1]
```diff
@@ -192,7 +193,7 @@ This parameter can only be used for listings, not for individual records. If you
 {{% notice note %}}
 A response obtained with "display" other than "full" can't be used in a **PUT** (update) request, because the `WebserviceRequest` class validation for fields is the same for **POST** (create) and **PUT** (update).
 
-This should be fixed in a near future with a *yet-to-come-pull-request* introducing the PATCH method!
+**PATCH** method allow you to use a partial display !
```
Comment (matthieu-rolland) [inline webservice/getting-started.md]:
```suggestion
The **PATCH** method allows using a partial display.
```

Also, doesn't this statement cancel the previous one (just above)? Then we should remove it too, right ?

**Review by matthieu-rolland — COMMENTED** [review]
thank you !

File: webservice/getting-started.md [hunk 2]
```diff
@@ -244,6 +245,10 @@ PrestaShop will take care of adding everything in the database, and will return
 
 To edit an existing resource: **GET** the full XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PUT HTTP request** with the whole XML file as a body content to the same URL again.
 
+### Partially update a resource
+
+To partially edit an existing resource: **GET** a part of the XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PATCH HTTP request** with the partial XML file as a body content to the same URL again.
```
Comment (matthieu-rolland) [inline webservice/getting-started.md]:
```suggestion
To partially edit an existing resource: **GET** a part of the XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PATCH HTTP request** with the partial XML file as the body content to the same URL again.
```

File: webservice/tutorials/prestashop-webservice-lib/setup-library.md [hunk 3]
```diff
@@ -97,6 +97,10 @@ Once the instance is created you can access the following methods:
 | edit()   | PUT               | UPDATE |
 | delete() | DELETE            | DELETE |
 
+{{% notice note %}}
+**PATCH** method is not yet managed with this library.
```
Comment (kpodemski) [inline webservice/tutorials/prestashop-webservice-lib/setup-library.md]:
```suggestion
**PATCH** method is not available in this library.
```

**Review by kpodemski — CHANGES_REQUESTED** [review]
Hey @PrestaEdit 

Nicely done! Could you follow Matthieu's suggestions? I've also added one :)

File: webservice/getting-started.md (hunk 1 above)
Comment (kpodemski) [reply webservice/getting-started.md]: @PrestaEdit you forgot about this one

Comment (PrestaEdit) [reply webservice/getting-started.md]: @kpodemski No, I was just away from keyboard between the two suggestions changes ;)

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thank you @PrestaEdit

**Pull request merged**